src/FactorGraphTypes.jl

```diff
@@ -58,6 +58,7 @@ mutable struct SolverParams <: DFG.AbstractParams
   N::Int
   multiproc::Bool
   logpath::String
+  algorithms::Vector{Symbol} # list of algorithms to run [:default] is mmisam
```

Comment:
think this is pretty cool and will be very useful for prototyping and dev work too -- great idea thanks!

Reply:
still a work in progress, but it seems to work